xbmc/video/VideoInfoTag.cpp

```diff
@@ -1062,6 +1062,17 @@ void CVideoInfoTag::SetUniqueId(std::string uniqueId)
   m_strUniqueId = Trim(std::move(uniqueId));
 }
 
+void CVideoInfoTag::SetUserrating(int userrating)
+{
+  //This value needs to be between 1-10
+  if (userrating < 1)
+    userrating = 1;
+  else if (userrating > 10)
+    userrating = 10;
+
```

Comment:
perhaps something like
userrating = std::max(userrating,1);
userrating = std::min(userrating,10);
?
Not sure what the most elegant solution is in C++.